src/components/Button/Button.tsx

```diff
@@ -0,0 +1,30 @@
+
+import React from "react";
+import './Button.css'
+
+interface ButtonProps {
+  submit: () => void;
+  buttonText?: string; 
+}
+
+
+
+export const Button = ({submit, buttonText}: ButtonProps) =>{
+return (
+  // modal__form-submit
+  <div className="modal__form-submit">
+          <button
+          // modal__form-submit__button
+            className="relative w-60 h-16 rounded-2xl border-4 border-[#cf554e] bg-[#cf554e]"
+            type="submit"
+          onClick={submit}
+          >
+          
+          {/* modal__form-submit__text */}
+            <p className="font-title text-white font-bold leading-6 text-base">{buttonText}</p>
```

Comment:
text-white is incorrect.
text-base is incorrect.

Reply:
To add on to Ruven's comments you can change them by using these colors right here. for example instead of `text-white` change it to `text-offWhite` according to the tailwind file. 
<img width="1440" alt="Screenshot 2024-09-11 at 10 44 53 PM" src="https://github.com/user-attachments/assets/018631c5-1c89-49f9-a5c3-4af167b9ce78">